lib/github_classroom/lti/membership_service.rb

```diff
@@ -72,7 +72,7 @@ def parse_membership(raw_data)
       def parse_membership_service(raw_data)
         begin
           json_membership = JSON.parse(raw_data)
-        rescue
+        rescue JSON::ParserError
           Rails.logger.error("raw_data: #{raw_data}")
           raise JSON::ParserError
```

Comment:
Could we add a comment about why we're not re-raising the error that was rescued? (talking to @jeffrafter it sounded like it's because we might cause problems with our logging applications if we log weird encoding).

Reply:
We are re-raising the error that was rescued in line 77.

Reply:
The error we're raising is a new error of the same type. To re-raise the exact error that was rescued, we'd need to set it to a variable with something like `rescue JSON::ParserError => e`.

Reply:
Made the changes 😄